ddpui/api/transform_api.py

```
@@ -751,3 +765,590 @@ def post_unlock_canvas(request, payload: LockCanvasRequestSchema):
+        raise HttpError(404, "dbt workspace not setup")
+
+    try:
+        canvas_node = CanvasNode.objects.get(uuid=node_uuid, orgdbt=orgdbt)
+
+        if canvas_node.dbtmodel:
+            dbtautomation_service.delete_dbt_model_in_project(canvas_node.dbtmodel)
+
+        canvas_node.delete()
+
+        logger.info(f"canvas node deleted successfully: {node_uuid}")
+        return {"success": 1}
```

Ruff (0.14.5)

Consider moving this statement to an `else` block

(TRY300)